examples/nas/multi-trial/mnist/search.py

```diff
@@ -106,7 +106,8 @@ def evaluate_model(model_cls):
     test_loader = DataLoader(MNIST('data/mnist', download=True, train=False, transform=transf), batch_size=64)
 
     device = torch.device('cuda') if torch.cuda.is_available() else torch.device('cpu')
-
+    
```

Comment:
Please remove space at end-of-line.

Reply:
:) I have removed it now, it seems to be vscode's fault.